types/eleventy/collection-item.d.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare global {
+  export interface EleventyCollectionItem {
```

Comment:
I'm happy with this contribution but is there like, definitelytyped types for Eleventy already?

Reply:
Haha, Rob actually asked the same thing. Unfortunately there is not, at least not one that I found (I checked for `@types/11ty` and `@types/eleventy`).